### PR TITLE
Jhalackova pcp container

### PR DESCRIPTION
**Pull request by lvicoun**
### PR creator: Description

Adding PCP container description for SLE Micro

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [X] SLE 15 SP3/openSUSE Leap 15.3/SLE Micro 5.2
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 SP0
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/art_admin_slemicro.xml [hunk 1]
```diff
@@ -1332,11 +1327,771 @@ After=etcd.service
...
+  -e PCP_SERVICES=pmlogger  \
+  -v pcp-archives:/var/log/pcp/pmlogger  \
+  registry.suse.com/suse/pcp:latest</screen>
+        </listitem>
+       </varlistentry>
+       <varlistentry>
+        <term>HOST_MOUNT</term>
+        <listitem>
+         <para>
+          is a path inside the container to the bind mount on the host. This
```
Comment (djoreilly) [inline xml/art_admin_slemicro.xml:1498]: `is the path inside the container to the bind mount of the host's root filesystem.`

File: xml/art_admin_slemicro.xml [hunk 2]
```diff
@@ -1332,11 +1327,771 @@ After=etcd.service
...
+<replaceable>CLIENT_HOSTNAME</replaceable>   y   n   PCP_ARCHIVE_DIR/<replaceable>CLIENT_HOSTNAME</replaceable>   -r -T24h10m -c config.default -v 100Mb</screen>
+      <para>
+       For example, for the <literal>slemicro_1</literal> host name, the line
+       should look as follows:
+      </para>
+<screen># local primary logger
+slemicro_1   y   n   PCP_ARCHIVE_DIR/slemicro_1   -r -T24h10m -c config.default -v 100Mb</screen>
+     </note>
+    </section>
+    <section xml:id="sec-config-default">
```
Comment (djoreilly) [inline xml/art_admin_slemicro.xml:1856]: I would remove this section. I'm not exactly sure how you would add metrics after the auto generation is done on container start. Anyway I don't see the need to manually add metrics.

**Review by djoreilly — CHANGES_REQUESTED** [review]

File: xml/art_admin_slemicro.xml [hunk 3]
```diff
@@ -1332,11 +1327,771 @@ After=etcd.service
...
+  </section>
+  <section xml:id="sec-collecting-metrics">
+   <title>Metrics management</title>
+   <section xml:id="sec-listing-metrics">
+    <title>Listing available performance metrics</title>
+    <para>
+     You can use the command <command>pminfo</command> to list metrics. For
+     example, to list all available performance metrics, run:
+    </para>
+<screen>&prompt.root;pminfo</screen>
```
[the remark this answers is not in the document]
Comment (lvicoun) [reply xml/art_admin_slemicro.xml:1900]: Or mention that the command should run inside the container?

**Review by djoreilly — APPROVED** [review]
lgtm - thanks!

File: xml/art_admin_slemicro.xml [hunk 4]
```diff
@@ -1332,11 +1327,760 @@ After=etcd.service
...
+     </para>
+<screen>-i 192.168.0.100</screen>
+    </section>
+  -->
+   </section>
+   <section xml:id="sec-pmlogger-configuration">
+    <title>Custom <command>pmlogger</command> configuration</title>
+    <para>
+     The custom configuration for the <command>pmlogger</command> is stored in
+     the following configuraiton files:
```
Comment (dariavladykina) [inline xml/art_admin_slemicro.xml:1715]:
```suggestion
     the following configuration files:
```

File: xml/art_admin_slemicro.xml [hunk 5]
```diff
@@ -1332,11 +1327,760 @@ After=etcd.service
...
+      frequency (default is 24 hours), and <command>pmlogger</command> options.
+      An example follows:
+     </para>
+<screen>
+# === VARIABLE ASSIGNMENTS ===
+#
+# DO NOT REMOVE OR EDIT THE FOLLOWING LINE
+$version=1.1
+
+# Uncomment one of the lines below to enable/disable compression behaviour
```
Comment (dariavladykina) [inline xml/art_admin_slemicro.xml:1809]:
```suggestion
# Uncomment one of the lines below to enable/disable compression behavior
```

File: xml/art_admin_slemicro.xml [hunk 6]
```diff
@@ -1332,11 +1327,760 @@ After=etcd.service
...
+#$PCP_COMPRESSAFTER=3
+#$PCP_COMPRESSAFTER=never
+    
+# === LOGGER CONTROL SPECIFICATIONS ===
+#   
+#Host           P?  S?  directory                       args
+
+# local primary logger
+LOCALHOSTNAME   y   n   PCP_ARCHIVE_DIR/LOCALHOSTNAME   -r -T24h10m -c config.dee
+fault -v 100Mb
```
Comment (dariavladykina) [inline xml/art_admin_slemicro.xml:1824]:
```suggestion
LOCALHOSTNAME   y   n   PCP_ARCHIVE_DIR/LOCALHOSTNAME   -r -T24h10m -c config.default -v 100Mb
```

**Review by dariavladykina — COMMENTED** [review]
Hi Jana, please take a look at some suggestions. Thanks!

**Pull request merged**